chapters/annotations.tex

```diff
@@ -2324,6 +2324,19 @@ \subsection{Licensing}\label{licensing}
 \end{example}
 
 
+\section{License Texts}\label{license-texts}
+
+For a top-level class the {\lstinline!annotation(License="modelica:/ModelicaLibraryName/Resources/Licenses/MyLicense.txt")!}\annotationindex{License}, gives a license text file for the class.
+The annotation may give a scalar value or an array of values.
```

Comment:
Here is a concrete proposal for addressing @otronarp's https://github.com/modelica/ModelicaSpecification/pull/2900#issuecomment-2022343060:
```suggestion
The annotation may give a scalar string or an array of strings, where each string is a URI referencing a license text file, see \cref{external-resources}.
```

Reply:
As I understand it this is the only minor issue to discuss after it was accepted.
We might either accept the PR without it and then have a separate PR for this proposal, or amend the PR (arguing that it is just a trivial change).

Reply:
I also raised one other concern previously:

> it is inconsistent in one location presumablely

I belive it can be easily addressed now.